pvr.hts/resources/language/resource.language.en_gb/strings.po

```diff
@@ -176,10 +200,48 @@ msgctxt "#30361"
 msgid "Record max once per day"
 msgstr ""
 
-#empty strings from id 30362 to 30399
+#empty strings from id 30362to 30364
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30365"
+msgid "1 day"
+msgstr ""
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30366"
+msgid "%d days"
+msgstr ""
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30367"
+msgid "1 week"
+msgstr ""
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30368"
+msgid "%d weeks"
+msgstr ""
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30369"
+msgid "1 month"
+msgstr ""
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30370"
+msgid "%d months"
+msgstr ""
+
+#empty strings from id 30371 to 30399
 
 msgctxt "#30400"
-msgid "Predictive tuning"
+msgid "Predictive tuning (experimental)"
```

Comment:
I wouldn't call this feature experimental anymore, it seems to work pretty well.